test/object-store/sync/flx_sync.cpp

```diff
@@ -1247,14 +1266,14 @@ TEST_CASE("flx: client reset", "[sync][flx][client reset][baas]") {
         };
 
     SECTION("Recover: schema indexes match in before and after states") {
+        config_local.sync_config->client_resync_mode = ClientResyncMode::Recover;
```

Comment:
are the changes to when `client_resync_mode` is set really needed?

Reply:
I was thinking about reverting those since they're not needed anymore.